<launch>
  <arg name="rect" default="true"/>
  <arg name="color_topic" value="image_raw" unless="$(arg rect)"/>
  <arg name="color_topic" value="image_rect_color" if="$(arg rect)"/>
  <node pkg="image_transport" type="republish" name="uncompress_color" args="compressed in:=color/camera/$(arg color_topic)_throttle raw out:=color/camera/$(arg color_topic)_throttle"/>
  <node pkg="image_transport" type="republish" name="uncompress_thermal" args="compressed in:=thermal/camera/image_raw_throttle raw out:=thermal/camera/image_raw_throttle"/>
</launch>
